CONTRIBUTING.md

```diff
@@ -0,0 +1,4 @@
+
+## Overview
+
+To contribute to this project, follow the rules from the general [CONTRIBUTING.md](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md) document in the `community` repository.
```

Comment:
The link follows to another document with another link that follows to concrete documentation, could we just ad link to the end file document?